sensor_msgs/RegionOfInterest roi
std_msgs/UInt8 score